lib/celluloid/calls.rb

```diff
@@ -23,9 +15,13 @@ def execute_block_on_receiver
     end
 
     def dispatch(obj)
-      check(obj)
-      _b = @block && @block.to_proc
-      obj.public_send(@method, *@arguments, &_b)
+      begin
+        check(obj)
+      rescue => error
+        raise AbortError, error
```

Comment:
This seems broken to me. It means that the `ArgumentErrors` below will only crash the receiver (but not the caller?)